prymer/variant.py

```diff
@@ -0,0 +1,259 @@
+import logging
```

Comment:
given `VariantLookup` is public, it'd be nice to have a module-level docstring with examples.  Would it be easy to port over the examples from the deleted module?

Reply:
I'll add it to our 4.0 tracker ... my thought it to iterate and get to a point where we're happy with all the APIs then spend a day on documentation and examples.